test/python/circuit/test_scheduled_circuit.py

```diff
@@ -159,21 +160,13 @@ def test_transpile_circuit_with_custom_instruction(self):
                               instruction_durations=[('bell', [0, 1], 1000)])
         self.assertEqual(scheduled.duration, 1500)
 
-    def test_transpile_delay_circuit_without_scheduling_method(self):
-        qc = QuantumCircuit(2)
-        qc.h(0)
-        qc.delay(500, 1)
-        qc.cx(0, 1)
-        transpiled = transpile(qc, backend=self.backend_with_dt)
-        self.assertEqual(transpiled.duration, 2132)
```

Comment:
How about updating this test like below instead of removing it?
```
    def test_transpile_delay_circuit_with_dt_but_without_scheduling_method(self):
        qc = QuantumCircuit(1)
        qc.delay(100, 0, unit='ns')
        transpiled = transpile(qc, backend=self.backend_with_dt)
        self.assertEqual(transpiled.duration, None)  # not scheduled
        self.assertEqual(transpiled.data[0][0].duration, 450)  # time unit is converted ns -> dt
```

Reply:
good suggestion